src/sentry/web/frontend/react_page.py

```diff
@@ -107,12 +107,17 @@ def handle_auth_required(self, request: Request, *args, **kwargs) -> HttpRespons
         # For normal users, let parent class handle (e.g. redirect to login page)
         return super().handle_auth_required(request, *args, **kwargs)
 
-    def handle(self, request: Request, organization, **kwargs) -> HttpResponse:
+    def handle(self, request: Request, organization: RpcOrganization, **kwargs) -> HttpResponse:
         if "project_id" in kwargs and request.GET.get("onboarding"):
-            project = Project.objects.filter(
-                organization=organization, slug=kwargs["project_id"]
-            ).first()
-            first_event_pending.send(project=project, user=request.user, sender=self)
+            project = project_service.get_by_slug(
```

Comment:
It seems like projects already belong to the `organization` object.  I don't think it is necessary to query the project again by slug, a simple filter on the objects inside `organization` should be sufficient?